Frends.Template/Frends.Echo.Execute/Frends.Echo.Execute.Tests/.env.example

```
@@ -0,0 +1,2 @@
+#Rename this file to .env to run tests locally
+FRENDS_SECRET_KEY="example"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)